yb-voyager/cmd/importSchema.go

```diff
@@ -191,6 +210,10 @@ func checkIfTargetSchemaExists(conn *pgx.Conn, targetSchema string) bool {
 	return fetchedSchema == targetSchema
 }
 
+func deferDoesNotExist(errString string) bool {
+	return strings.Contains(errString, "does not exist")
```

Comment:
What about the following error:
```
ERROR: there is no unique constraint matching given keys for referenced table "export_setting_histories" (SQLSTATE 42830) – Issue no. 
```

Reply:
this is different scenarios, not required for deferred list. Covered by @priyanshi-yb in https://github.com/yugabyte/yb-voyager/pull/616